.travis.yml

```diff
@@ -36,8 +36,11 @@ matrix:
 # command to install dependencies
 install:
     - python --version
-    - pip install -r tests/requirements.txt
-    - python setup.py install
+    - export PUBS_TESTS_MODE=ONLINE
 
 # command to run tests
-script: python -m unittest discover
+script:
+  - export PUBS_TESTS_MODE=ONLINE
+  - python setup.py test
+  - export PUBS_TESTS_MODE=MOCK
+  - python setup.py test
```

Comment:
Maybe:
```yaml
script: 
  - PUBS_TESTS_MODE=ONLINE python setup.py test
  - PUBS_TESTS_MODE=MOCK python setup.py test
```

Reply:
Ok, I went with:
```
script:
  - PUBS_TESTS_MODE=MOCK python setup.py test
  - PUBS_TESTS_MODE=COLLECT python setup.py test
```
to test the collect code.